eatin-server/migrate/V8__add_comment_and_is_upload_to_userrecipes.pgsql

```diff
@@ -0,0 +1,2 @@
+alter table userrecipes add column is_uploaded boolean not null default false;
+alter table userrecipes add column given_comment varchar(100) not null default '';
```

Comment:
you should change this column type from `varchar(100)` to `text` so we will be able to save comments without worrying about text size

Reply:
I need to add another migrate file? cuz i cant change it just as it is